File: src/main/kotlin/model/Game.kt
package model

import org.eclipse.jetty.websocket.api.Session
import java.util.HashMap

class Game(
    val targets: Collection<Target>,
    cursors: HashMap<Session, Cursor>,
    session: Session? = null
) {
    val cursors: Collection<Cursor> = cursors.filterKeys { session !== it }.values
}

File: client/src/models/Game.ts
import Target from "models/Target";
import Cursor from "models/Cursor";

export default class Game {
	targets: Array<Target>;
	cursors: Array<Cursor>;

	constructor(game: any) {
		this.targets = [];
		game.targets.forEach(target =>
			this.targets.push(new Target(target))
		)
		this.cursors = [];
		game.cursors.forEach(cursor =>
			this.cursors.push(new Cursor(cursor))
		)
	}
}